models/person.rb

```diff
@@ -24,6 +26,14 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def books
+    rentals.map(&:book)
+  end
+
+  def add_rental(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Please rewrite this `add_rental` method bearing in mind that the `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(book, date)
  # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
  Rental.new(date, book, self)
end
```